libsysbox/syscont/spec.go

```diff
@@ -807,17 +806,6 @@ func sysMgrSetupMounts(mgr *sysbox.Mgr, spec *specs.Spec, rootfsUidShiftType sh.
 		_, isSpecialDir := specialDirMap[m.Destination]
 
 		if m.Type == "bind" && isSpecialDir {
-
```

Comment:
Yes it's a good point; IIRC the sysbox-runc -> sysbox-mgr gRPC latency is minimal; most of the delay in sysbox-runc comes from other places (e.g., reading & parsing mountinfo from the kernel). 

I left the prep mounts step in there to catch the situation where the same host dir is mounted into the `/var/lib/docker` of two or more containers, since that could create all sorts of problems. Only sysbox-mgr can track that (sysbox-runc can't) so I decided the cost/benefit was worth it.

Reply:
Understood.